src/components/30-organisms/footer/index.scss

```diff
@@ -216,8 +216,8 @@
           border-bottom: none;
         }
 
-        @include breakpoint($mediaquery-xs-up) {
-          @include typo(primary, default, default);
+        @include breakpoint($mediaquery-sm-up) {
```

Comment:
You changed the behaviour from xs to sm. Because the breakpoint also changed or a mistake?

Reply:
done